### PR TITLE
version bump: erlang-19.0.3 & 18.3.4.3

### PR DESCRIPTION
**Pull request by joedevivo**

### CODE REVIEW

**Comment by tianon** [comment]
Sweet https://github.com/c0b/docker-erlang-otp/pull/18#issuecomment-238990442 (PR created at maintainer's request)

```diff
diff --git a/erlang_17-onbuild/Dockerfile b/erlang_17-onbuild/Dockerfile
new file mode 100644
index 0000000..0fa0ae3
--- /dev/null
+++ b/erlang_17-onbuild/Dockerfile
@@ -0,0 +1 @@
+FROM erlang:17
diff --git a/erlang_18-slim/Dockerfile b/erlang_18-slim/Dockerfile
index 825c813..e721c73 100644
--- a/erlang_18-slim/Dockerfile
+++ b/erlang_18-slim/Dockerfile
@@ -1,12 +1,12 @@
 FROM debian:jessie
 
-ENV OTP_VERSION="18.3.4.1"
+ENV OTP_VERSION="18.3.4.3"
 
 # We'll install the build dependencies, and purge them on the last step to make
 # sure our final image contains only what we've just built:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-$OTP_VERSION.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="3b9852083f3e5cf0668757af76833547233b8e67f4c28b51a6f49d4625bb23de" \
+	&& OTP_DOWNLOAD_SHA256="9cfb0315ae2650c3b1bb6f66f430753593f014698fd6105ef19cfd8eb90f72fe" \
 	&& runtimeDeps=' \
 		libodbc1 \
 		libssl1.0.0 \
diff --git a/erlang_18/Dockerfile b/erlang_18/Dockerfile
index 6dcfcdc..2e241e1 100644
--- a/erlang_18/Dockerfile
+++ b/erlang_18/Dockerfile
@@ -1,12 +1,12 @@
 FROM buildpack-deps:jessie
 
-ENV OTP_VERSION="18.3.4.1"
+ENV OTP_VERSION="18.3.4.3"
 
 # We'll install the build dependencies for erlang-odbc along with the erlang
 # build process:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-$OTP_VERSION.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="3b9852083f3e5cf0668757af76833547233b8e67f4c28b51a6f49d4625bb23de" \
+	&& OTP_DOWNLOAD_SHA256="9cfb0315ae2650c3b1bb6f66f430753593f014698fd6105ef19cfd8eb90f72fe" \
 	&& runtimeDeps='libodbc1 \
 			libsctp1' \
 	&& buildDeps='unixodbc-dev \
@@ -61,4 +61,3 @@ RUN set -xe \
 	&& HOME=$PWD ./bootstrap \
 	&& install -v ./rebar3 /usr/local/bin/ \
 	&& rm -rf /usr/src/rebar3-src
-
diff --git a/erlang_latest/Dockerfile b/erlang_latest/Dockerfile
index 998c940..c377fb1 100644
--- a/erlang_latest/Dockerfile
+++ b/erlang_latest/Dockerfile
@@ -1,12 +1,12 @@
 FROM buildpack-deps:jessie
 
-ENV OTP_VERSION="19.0.1"
+ENV OTP_VERSION="19.0.3"
 
 # We'll install the build dependencies for erlang-odbc along with the erlang
 # build process:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="b425c6c009f1b75bf478306e15f948c3031d1a41ee7f7e5ac6845cc1b740fd44" \
+	&& OTP_DOWNLOAD_SHA256="feb8c5fe111087443c90ce189b979b0288ba358ebffb9be18a99cc69719cacc8" \
 	&& runtimeDeps='libodbc1 \
 			libsctp1' \
 	&& buildDeps='unixodbc-dev \
diff --git a/erlang_slim/Dockerfile b/erlang_slim/Dockerfile
index 6cf6e54..ee0765d 100644
--- a/erlang_slim/Dockerfile
+++ b/erlang_slim/Dockerfile
@@ -1,12 +1,12 @@
 FROM debian:jessie
 
-ENV OTP_VERSION="19.0.1"
+ENV OTP_VERSION="19.0.3"
 
 # We'll install the build dependencies, and purge them on the last step to make
 # sure our final image contains only what we've just built:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION##*@}.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="b425c6c009f1b75bf478306e15f948c3031d1a41ee7f7e5ac6845cc1b740fd44" \
+	&& OTP_DOWNLOAD_SHA256="feb8c5fe111087443c90ce189b979b0288ba358ebffb9be18a99cc69719cacc8" \
 	&& runtimeDeps=' \
 		libodbc1 \
 		libssl1.0.0 \
```

File: library/erlang [hunk 1]
```diff
@@ -1,30 +1,30 @@
 # this file is generated via https://github.com/c0b/docker-erlang-otp/blob/8e61adc781b30f61cc017ffa84ec1ad8fc7d6754/generate-stackbrew-library.sh
 
-Maintainers: Mr C0B <denc716@gmail.com> (@c0b)
+Maintainers: denc716@gmail.com (@c0b)
```
Comment (tianon) [inline library/erlang]: I've sent https://github.com/c0b/docker-erlang-otp/pull/19 upstream to take care of this change.

File: library/erlang [hunk 2]
```diff
@@ -34,3 +34,7 @@ Directory: 17
 Tags: 17.5.6.9-slim, 17.5.6-slim, 17.5-slim, 17-slim
 GitCommit: ea32d5f6f1735f9f55bee04b112166da96eb9c73
 Directory: 17/slim
+
+Tags: 17.5.6.9-onbuild, 17.5.6-onbuild, 17.5-onbuild, 17-onbuild
+GitCommit: dbcb2dd16843db0663d10278f2a13cc70a840494
+Directory: 17/onbuild
```
Comment (tianon) [inline library/erlang]: I've sent https://github.com/c0b/docker-erlang-otp/pull/20 upstream to take care of this change.

**Comment by c0b** [comment]
both PRs from @tianon are merged in c0b/docker-erlang-otp#21 @joedevivo please just refresh this PR

**Comment by joedevivo** [comment]
Refreshed.

**Comment by yosifkit** [comment]
LGTM, Build test of #2042; 51a743e (`erlang`):

```console

$ bashbrew build erlang:19.0.3
Building bashbrew/cache:b557936820fd41ccf1b0f030f7fa1345aa2fbc7c59d8660ca95acad83c368031 (erlang:19.0.3)
Tagging erlang:19.0.3
Tagging erlang:19.0
Tagging erlang:19
Tagging erlang:latest

$ test/run.sh erlang:19.0.3
testing erlang:19.0.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'erlang-hello-world' [5/5]...passed


$ bashbrew build erlang:19.0.3-slim
Building bashbrew/cache:f12702621deed35a50486b76b244334a1315aecec94cda458cf2562cff7fe0ed (erlang:19.0.3-slim)
Tagging erlang:19.0.3-slim
Tagging erlang:19.0-slim
Tagging erlang:19-slim
Tagging erlang:slim

$ test/run.sh erlang:19.0.3-slim
testing erlang:19.0.3-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'erlang-hello-world' [5/5]...passed


$ bashbrew build erlang:19.0.3-onbuild
Building bashbrew/cache:527fd7a95479aceb7358268d35a9e1a5740841c080844586a3fd9ea5514f70fc (erlang:19.0.3-onbuild)
Tagging erlang:19.0.3-onbuild
Tagging erlang:19.0-onbuild
Tagging erlang:19-onbuild
Tagging erlang:onbuild

$ test/run.sh erlang:19.0.3-onbuild
testing erlang:19.0.3-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build erlang:18.3.4.3
Building bashbrew/cache:80095a80c621a3ad806c7a978e305a658079235010613dcec28d78cd48114d6f (erlang:18.3.4.3)
Tagging erlang:18.3.4.3
Tagging erlang:18.3.4
Tagging erlang:18.3
Tagging erlang:18

$ test/run.sh erlang:18.3.4.3
testing erlang:18.3.4.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'erlang-hello-world' [5/5]...passed


$ bashbrew build erlang:18.3.4.3-slim
Building bashbrew/cache:355f334b8e38a9d41893412fba087c87fac3dc399331fddf9079eaa4e9ac2001 (erlang:18.3.4.3-slim)
Tagging erlang:18.3.4.3-slim
Tagging erlang:18.3.4-slim
Tagging erlang:18.3-slim
Tagging erlang:18-slim

$ test/run.sh erlang:18.3.4.3-slim
testing erlang:18.3.4.3-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'erlang-hello-world' [5/5]...passed


$ bashbrew build erlang:18.3.4.3-onbuild
Building bashbrew/cache:8af23667483853294b983d94b0f835e33d655a808bee8bcee33cffefc87f55e1 (erlang:18.3.4.3-onbuild)
Tagging erlang:18.3.4.3-onbuild
Tagging erlang:18.3.4-onbuild
Tagging erlang:18.3-onbuild
Tagging erlang:18-onbuild

$ test/run.sh erlang:18.3.4.3-onbuild
testing erlang:18.3.4.3-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build erlang:17.5.6.9
Using bashbrew/cache:92b4370db28443ea4570c452d8c39cf0f8d60b8dacb475ab7db908aeb259e7aa (erlang:17.5.6.9)
Tagging erlang:17.5.6.9
Tagging erlang:17.5.6
Tagging erlang:17.5
Tagging erlang:17

$ test/run.sh erlang:17.5.6.9
testing erlang:17.5.6.9
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'erlang-hello-world' [5/5]...passed


$ bashbrew build erlang:17.5.6.9-slim
Using bashbrew/cache:be19b187d282ee3feb82ef63fb866063acc987e9d1c8f70798f3c87e957214a8 (erlang:17.5.6.9-slim)
Tagging erlang:17.5.6.9-slim
Tagging erlang:17.5.6-slim
Tagging erlang:17.5-slim
Tagging erlang:17-slim

$ test/run.sh erlang:17.5.6.9-slim
testing erlang:17.5.6.9-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'erlang-hello-world' [5/5]...passed

```

**Pull request merged**